editor/src/components/canvas/ui/floating-insert-menu.tsx

```diff
@@ -126,6 +128,26 @@ function useGetInsertableComponents(): InsertableComponentFlatList {
   return insertableComponents
 }
 
+function getIsFlexBasedOnName_KILLME_EXPERIMENTAL(name: JSXElementName): boolean {
```

Comment:
Can you add a ticket for removing these please

Reply:
big time

Reply:
added ticket https://github.com/concrete-utopia/utopia/issues/1580